### PR TITLE
feat: update seen_at pat column

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-451/update-last-seen-column-for-pats

### CODE REVIEW

**Review by gastonfournier — COMMENTED** [review]
LGTM! Let's focus on the other PR https://github.com/Unleash/unleash/pull/2514 and if everything is fine with that one, this one will be as well

File: src/lib/services/user-service.ts [hunk 1]
```diff
@@ -426,6 +442,30 @@ class UserService {
...
+    async updateLastSeen(): Promise<void> {
+        if (this.lastSeenSecrets.size > 0) {
+            const toStore = [...this.lastSeenSecrets];
+            this.lastSeenSecrets = new Set<string>();
+            await this.store.markSeenAt(toStore);
+        }
+
+        this.seenTimer = setTimeout(
+            async () => this.updateLastSeen(),
+            minutesToMilliseconds(3),
```
Comment (sighphyre) [inline src/lib/services/user-service.ts:455]: I don't have a problem with it, just curious, why 3? It feels like such a carefully chosen number

**Review by sighphyre — APPROVED** [review]
LGTM

File: src/lib/services/user-service.ts (hunk 1 above)
Comment (nunogois) [reply src/lib/services/user-service.ts:455]: It's just so it's aligned with the other PR: https://github.com/Unleash/unleash/pull/2514#discussion_r1033938370

Comment (sighphyre) [reply src/lib/services/user-service.ts:455]: Ah!

**Pull request merged**